programs/survival/show_bees_count.sc

```diff
@@ -0,0 +1,13 @@
+// shows the count of bees on that specific block
+// right click while sneaking on a beehive or beenest
+
+__command()->null;

```

Comment:
You only need to add `__command()` if you want the app to actually have a command tree. If it doesn't use player commands, you can just skip it.

Reply:
Same as above